docker/docker.ts

```diff
@@ -135,6 +137,28 @@ export async function buildAndPushImageAsync(
     logResource: pulumi.Resource,
     connectToRegistry?: () => Promise<Registry>): Promise<string> {
 
+    // Give an initial message indicating what we're about to do.  That way, if anything
+    // takes a while, the user has an idea about what's going on.
+    logEphemeral("Starting docker build and push...", logResource);
+
+    const result = await buildAndPushImageWorkerAsync(
+        imageName, pathOrBuild, repositoryUrl, logResource, connectToRegistry);
+
+    // If we got here, then building/pushing didn't throw any errors.  Update the status bar
+    // indicating that things worked properly.  That way, the info bar isn't stuck showing the very
+    // last thing printed by some subcommand we launched.
+    logEphemeral("Successfully pushed to docker", logResource);
+
+    return result;
```

Comment:
We now print along ephemeral messages around the docker process (and around individual docker commands) to help give an idea about what's going on. These are all ephemeral, so htey only show up in the status column, and not at the end of hte run.

Reply:
awesome!